packages/addons-v5/commands/addons/open.js

```diff
@@ -3,13 +3,14 @@
 let cli = require('heroku-cli-util')
 let co = require('co')
 let fs = require('fs')
+let os = require('os')
 
 function open (url) {
   cli.log(`Opening ${cli.color.cyan(url)}...`)
   return cli.open(url)
 }
 
-const ssoPath = '/tmp/heroku-sso.html'
+const ssoPath = os.tmpdir() + '/heroku-sso.html'
```

Comment:
can you add an import for `path = require('path')` at the top of the page and use `path.join(os.tmpdir(), 'heroku-sso.html') instead? this will help keep the path separators normalized between platforms.

Reply:
Nice! Made this change, thanks!